elasticsearch/run.sh

```diff
@@ -100,8 +104,30 @@ rm -f $LOG_FILE
 exit 1
 }
 
+seed_searchguard(){
+    /usr/share/elasticsearch/plugins/search-guard-2/tools/sgadmin.sh \
+        -cd ${HOME}/sgconfig \
+        -i .searchguard.${HOSTNAME} \
```

Comment:
gotcha, i was wondering where we provided the `Deletes SG index on start up if exists` logic is why... (taken from linked PR)